dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -59,10 +59,7 @@ def get_strategy(choice, model, ckpt_dir_path):
     return strategy
 
 
-def main(project: str,
```

Comment:
nit: As much as possible it's probably better to try to leave os.environ calls inside of main or the `if __name__ == '__main__'` block, and pass parameters to functions like `get_strategy`. It's not a major difference, but it ensures they can be cleanly reused outside of this code if needed

Reply:
+1

Reply:
Done.